pidgin-libnotify-0.14-libnotify-0.7.patch

```diff
@@ -0,0 +1,18 @@
+--- src/pidgin-libnotify.c
++++ src/pidgin-libnotify.c
+@@ -286,7 +286,15 @@
+ 		g_free (tr_body);
+ 		return;
+ 	}
++#ifdef NOTIFY_CHECK_VERSION
++#if NOTIFY_CHECK_VERSION (0, 7, 0)
++	notification = notify_notification_new (title, tr_body, NULL);
++#else
+ 	notification = notify_notification_new (title, tr_body, NULL, NULL);
++#endif
++#else
++	notification = notify_notification_new (title, tr_body, NULL, NULL);
```

Comment:
1. This patch is silly:

```c
#if !defined(NOTIFY_CHECK_VERSION) || !NOTIFY_CHECK_VERSION(0, 7, 0)
    notification = notify_notification_new (title, tr_body, NULL, NULL);
#else
    notification = notify_notification_new (title, tr_body, NULL, NULL);
#endif
```

2. Is this really not upstream, this version is ancient?

Reply:
It does not build without this patch. And pidgin-libnotify is quite old yes.